### DISCUSSION TITLE
Error on 3rd Section of course

### DISCUSSION THREAD

**Discussion by MisterMadrid** [Cyfrin/foundry-full-course-cu#249] [General]
Hi, I made it through the first 2 segments of the course, and, obviously, still having some trouble grasping concepts but I got the first contract deployed through the injected metamask. My issue is in Lessson 3 whenever I try to deploy or Compile the Storage Factory,it has the error, SimpleStorage.sol not found highlighted in orange. I deploy the first contract, but the second contract doesn't seem to be interacting with the first one or basically has no knowledge of its existence.. any ideas?

**Comment by aarav1656** [comment]
Share code or screenshot
